### ISSUE TITLE
Normalization of input images

### ISSUE THREAD

**Issue by almaan** [ozanciga/self-supervised-histopathology#18]
Hi,

thanks for sharing this, very appreciated. Me and some people in my team are interested in using the pre-trained model to generate representations of some of our images. Looking through the paper and the repo, it wasn't clear to us if there's any form of normalization being applied to the images before being passed to the model.

Judging from:
`images = torch.rand((10, 3, 224, 224), device='cuda')`

in your "Starter code" it seems like you just expect the image values to be in [0,1], but except from that, no other transforms are applied (e.g., subtraction by mean and division by standard deviation etc.).

If you could shed any light on this, it would be amazing.

Again, thank you for sharing!

**Comment by alexkrz** [comment]
See #2
